src/useCases/getCompanies.ts

```diff
@@ -0,0 +1,36 @@
+import { prisma } from '@/lib/prisma'
+import type { Company } from '@prisma/client'
+
+interface GetCompaniesResult {
+  status: number
+  success: boolean
+  data?: Company[]
+  message?: string
+}
+
+export const getCompanies = async (): Promise<GetCompaniesResult> => {
+  try {
+    const companies = await prisma.company.findMany({
+      select: {
```

Comment:
We don't need this `select` if we want to request all fields from the table. Please replace it with `const companies = await prisma.company.findMany`